lighthouse-plugin-publisher-ads/computed/ad-lantern-metric.js

```diff
@@ -77,21 +77,25 @@ function isLongTask(cpuNode) {
  */
 function addEdges(graph) {
   /** @type {NetworkNode[]} */ const adRequestNodes = [];
+  /** @type {NetworkNode[]} */ const gptJsNodes = [];
   graph.traverse((node) => {
-    if (node.type === BaseNode.TYPES.NETWORK && isGptAdRequest(node.record)) {
+    if (node.type !== BaseNode.TYPES.NETWORK) {
+      return;
+    }
+    if (isGptTag(node.record.url) && node.record.resourceType === 'Script') {
+      gptJsNodes.push(node);
+    } else if (isGptAdRequest(node.record)) {
       adRequestNodes.push(node);
     }
   });
   graph.traverse((node) => {
     if (node.type !== BaseNode.TYPES.NETWORK) {
       return;
     }
-    if (isBidRelatedRequest(node.record)) {
```

Comment:
Why remove this edge?

Reply:
Woops, didn't mean to check that in. Will revert